WIP-CHANGELOG.md

```diff
@@ -8,5 +8,6 @@
 ### Fixed
 
 ### Added
+- Option to configure alerts for growing primary shards in OpenSearch
```

Comment:
I'm fine just writing `indices` instead of `primary shards` as not to confuse people
```suggestion
- Option to configure alerts for growing primary shards in OpenSearch
```